core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -185,12 +185,13 @@ object Boot extends App with LazyLogging {
 
     val maxActiveWorkflowsTotal = conf.getInt("executionservice.maxActiveWorkflowsPerServer") * executionServiceServers.size
     val maxActiveWorkflowsPerUser = maxActiveWorkflowsTotal / conf.getInt("executionservice.activeWorkflowHogFactor")
+    val useWorkflowCollectionField = conf.getBoolean("executionservice.useWorkflowCollectionField")
```

Comment:
probably good to doublecheck whether we might want to have _both_ at some point during the migration, in which case this won't work

Reply:
I could add 2 separate flags instead